node/pegnet/addresses.go

```diff
@@ -183,6 +183,53 @@ func (p *Pegnet) SelectBalance(adr *factom.FAAddress, ticker fat2.PTicker) (uint
 	return balance, nil
 }
 
+type BalancePair struct {
+	Address *factom.FAAddress
+	Balance uint64
+}
+type BalancesPair struct {
+	Address  *factom.FAAddress
+	Balances []uint64
+}
+
+// SelectRichList returns the balance of all addresses for a given ticker
+func (p *Pegnet) SelectRichList(ticker fat2.PTicker, count int) ([]BalancePair, error) {
+	if ticker <= fat2.PTickerInvalid || fat2.PTickerMax <= ticker {
+		return nil, fmt.Errorf("invalid token type")
+	}
+	if count < 1 {
+		return nil, fmt.Errorf("invalid count")
+	}
+	var res []BalancePair
```

Comment:
If we replace this with `res := make([]BalancePair, 0)` we will get an empty list vs a nil. 

I think the empty list would be better behavior. What do you think?

Reply:
nobody should be calling this function with a count of 0, if they do it's likely an error so i'm leaning on the side of returning an error